libs/admin/src/lib/components/users/users.component.scss

```
@@ -0,0 +1,32 @@
+@use "../../styles/variables";
+
+::ng-deep .workgroups-tooltip {
+  font-size: 12px;
+}
+
+.users {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+}
+
+.users__header {
+  display: flex;
+  justify-content: start;
```

Prefer `flex-start` and `flex-end` over `start` and `end` in flex layouts, as the latter two are not as widely supported.
```suggestion
  justify-content: flex-start;
```